react/src/services/utils.ts

```diff
@@ -225,6 +225,8 @@ export const decimalToPercent = (decimal: number) => parseFloat((decimal * 1000
 
 export const isBeforeToday = (d: Date) => moment(d).isBefore(moment())
 
+export const isNotCurrentYear = (d: Date) => moment(d).isBefore(moment().subtract(1, 'years'))
```

Comment:
This new method checks that a date is before today's date minus one year. Found the `subtract` method through the documentation for moment.js: https://momentjs.com/docs/#/manipulating/subtract/